notebooks/galaxy_redshift/redshift_fitting.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>Providing a description of the inferences that can be made from the plot would make it more accessible to all readers, including those who use screen readers. This description should include information on the relationships between variables, patterns in the data, etc. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Adding descriptions to all plots.